extension-manifest-v2/webpack.config.js

```diff
@@ -42,7 +44,7 @@ const AUTOCONSENT_DIR = path.resolve(__dirname, 'app/autoconsent');
 const RM = (process.platform === 'win32') ? 'powershell remove-item' : 'rm';
 
 module.exports = {
-	devtool: 'none', // source-maps
+	// devtool: 'source-maps',
```

Comment:
Clean commented line.

Reply:
it is intentional, we want to generate source-maps for Sentry. Better to have a comment that to check for it in WebPack docs.